### PR TITLE
Add LJHarb to LJHarb's packages

### PR DESCRIPTION
**Pull request by ljharb**
This adds me as an owner on my own packages.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

per https://github.com/DefinitelyTyped/DefinitelyTyped/pull/41988#issuecomment-582674757

### CODE REVIEW

**Comment by ljharb** [comment]
k, i think that's all of the packages i maintain, but i didn't exhaustively look through the list.

It'd be kind of nice if DT automatically looked up the npm owners and pinged them :-)

**Review by Richienb — APPROVED** [review]

**Review by ExE-Boss — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -33,6 +33,5 @@
         }
     },
```
Comment (ExE-Boss) [inline package.json]:
```suggestion
```

File: package.json [hunk 2]
```diff
@@ -33,6 +33,5 @@
...
     "dependencies": {
```
Comment (ExE-Boss) [inline package.json]:
```suggestion
```

**Review by ExE-Boss — COMMENTED** [review]
The&nbsp;`package.json#dependencies`&nbsp;object shouldn’t&nbsp;be&nbsp;present if&nbsp;empty, see&nbsp;<https://github.com/DefinitelyTyped/DefinitelyTyped/pull/40009>&nbsp;and&nbsp;<https://github.com/DefinitelyTyped/DefinitelyTyped/pull/42147>.

**Review by VitorLuizC — APPROVED** [review]

**Review by slavik57 — APPROVED** [review]

**Review by wbhob — APPROVED** [review]

**Comment by ljharb** [comment]
Not sure these needed publishing for just a comment change ;-)

**Comment by jessetrinity** [comment]
Comment only PRs are probably rare enough that none of us have noticed

**Pull request merged**